libraries/fwd/computeFwd/compute_fwd.cpp

```diff
@@ -2107,12 +2109,12 @@ void ComputeFwd::initFwd()
         if (!coilfile) {
             return;
         }
+        FREE_41(coilfile);
```

Comment:
Why are we moving free coilfile to before we use it?

Reply:
Because on my quest for not leaking I forgot we're supposed to actually compute something, eventually...

Reply:
This code was updated and the actual commits lost in a rebase.